<!-- -*- mode: XML -*- -->

<launch>
  <arg name="rviz" default="true" />
  
 <arg name="map_config_path" default = "$(find monolio)/config/kitti_loop_config_00.yaml" />
  <node pkg = "monolio" type = "monolio_map_node" name ="monolio_map" output = "screen">
        <param name = "map_config_file" type= "string" value ="$(arg loop_config_path)"/>
  </node> 
</launch>
